version.cmake

```diff
@@ -4,8 +4,8 @@
 ## NOTE: when updating the version info please make the corresponding changes in pom.xml 
 ###
 set ( HPCC_MAJOR 0 )
-set ( HPCC_MINOR 2 )
-set ( HPCC_POINT 6 )
+set ( HPCC_MINOR 3 )
+set ( HPCC_POINT 0 )
```

Comment:
It's showing up in this commit, but this change was actually done in a different commit. Will rebase branch